src/components/Notification/Wrappers/DrawerWrapper.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
```

Comment:
Can we name this `NotificationDrawerWrapper` to be consistent? Same for the rest of the files under Wrappers.
NotificationDrawerPanelWrapper, NotificationDrawerStatefulWrapper etc...

Also, should we pull this all under NotificationDrawer? i.e. `NotificationDrawer.Wrapper`, `NotificationDrawer.StatefulWrapper`, ...

Reply:
Not sure the wrappers should be under, your call.